HIRS_AttestationCA/src/main/java/hirs/attestationca/persist/validation/PcrValidator.java

```diff
@@ -163,36 +163,36 @@ public StringBuilder validatePcrs(final String[] storedPcrs,
      * will ignore certin PCRs, Event Types and Event Variables present.
      *
      * @param tcgMeasurementLog Measurement log from the client
-     * @param eventValueMap     The events stored as baseline to compare
+     * @param eventLogRecords   The events stored as baseline to compare
      * @param policySettings    db entity that holds all of policy
      * @return the events that didn't pass
      */
     public List<TpmPcrEvent> validateTpmEvents(final TCGEventLog tcgMeasurementLog,
-                                               final Map<String, ReferenceDigestValue> eventValueMap,
+                                               final Map<String, ReferenceDigestValue> eventLogRecords,
                                                final PolicySettings policySettings) {
         List<TpmPcrEvent> tpmPcrEvents = new LinkedList<>();
         for (TpmPcrEvent tpe : tcgMeasurementLog.getEventList()) {
             if (policySettings.isIgnoreImaEnabled() && tpe.getPcrIndex() == IMA_PCR) {
-                log.info("IMA Ignored -> {}", tpe);
+                log.info(String.format("IMA Ignored -> %s", tpe));
```

Comment:
I've noticed that you've changed the parameterized log messages to string format messages throughout the three files. Is this the standard that we are supposed to follow or is there another reason for this change?

Reply:
This change was automatically applied by checkstyles so I assumed that was the standard we were following.  I can look into logging best practices to see if there's any reason to prefer one over the other.

Reply:
No worries! I wasn't trying to be nit-picky. I wasn't entirely sure if this was the standard we wanted to stick to or if parameterized log messages was the way to go. I saw that the Intellij IDE suggested that as the "preferred" option; however I do not know what the team believes is best.